### DISCUSSION TITLE
notes/2024-06-18-stick-with-it/

### DISCUSSION THREAD

**Discussion by giscus** [asboyer/asboyer.github.io#22] [Comments]
# notes/2024-06-18-stick-with-it/

Hi, I'm Andrew Boyer and this is my personal website, asboyer.com. Here you can find more about what I’m up to and what I find interesting. 

https://www.asboyer.com/notes/2024-06-18-stick-with-it/

**Comment by asboyer** [comment]
Comments weren't working before, now they are!

**Comment by BWang49** [comment]
Awesome skill set—pushing through the discomfort even though it’s hard and you feel like giving up is where the real reward lies! Applies to so many things in life as well.

**Reply by asboyer** [reply]
Thank you, glad you found this helpful.
